checkout-example-advanced/src/main/resources/static/adyenImplementation.js

```diff
@@ -1,105 +1,143 @@
 const clientKey = document.getElementById("clientKey").innerHTML;
-const type = document.getElementById("type").innerHTML;
+const { AdyenCheckout, Dropin } = window.AdyenWeb;
```

Comment:
We used to have a single file `adyenImplementation.js` for drop-in and components, but now we probably want to create individual files (`dropin.js`, `ideal.js`, `card.js`, etc..) I think.
Similarly to the Vue implementation we can keep those separated.

Reply:
Will do that